src/families/bitcoin/ScreenEditCustomFees.js

```diff
@@ -0,0 +1,140 @@
+/* @flow */
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import React, { useState, useCallback } from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { Keyboard, StyleSheet, TextInput, View } from "react-native";
+import SafeAreaView from "react-native-safe-area-view";
```

Comment:
prefer the usage of react-native's SafeAreaView, this one tends to make the screen jump